src/modules/endpoint/sagas/__tests__/loadCityContent.spec.js

```diff
@@ -0,0 +1,279 @@
+// @flow
+
+import DefaultDataContainer from '../../DefaultDataContainer'
+import RNFetchBlob from '../../../../__mocks__/rn-fetch-blob'
+import { expectSaga } from 'redux-saga-test-plan'
+import loadCityContent from '../loadCityContent'
+import AppSettings from '../../../settings/AppSettings'
+import { ContentLoadCriterion } from '../../ContentLoadCriterion'
+import type { DataContainer } from '../../DataContainer'
+import CategoriesMapModelBuilder from '../../../../testing/builder/CategoriesMapModelBuilder'
+import LanguageModelBuilder from '../../../../testing/builder/LanguageModelBuilder'
+import CityModelBuilder from '../../../../testing/builder/CityModelBuilder'
+import moment from 'moment-timezone'
+import EventModelBuilder from '../../../../testing/builder/EventModelBuilder'
+import AsyncStorage from '@react-native-community/async-storage'
+import fetchResourceCache from '../fetchResourceCache'
+import NetInfo from '@react-native-community/netinfo'
+
+jest.mock('@react-native-community/async-storage')
+jest.mock('@react-native-community/netinfo')
+jest.mock('rn-fetch-blob')
+jest.mock('../fetchResourceCache')
+jest.mock('../loadCategories')
+jest.mock('../loadEvents')
+jest.mock('../loadCities')
+jest.mock('../loadLanguages')
+
+const prepareDataContainer = async (dataContainer: DataContainer, city: string, language: string) => {
+  const categoriesBuilder = new CategoriesMapModelBuilder(2, 2)
+  const eventsBuilder = new EventModelBuilder('loadCityContent-events', 2)
+
+  const categories = categoriesBuilder.build()
+  const cities = new CityModelBuilder(1).build()
+  const languages = new LanguageModelBuilder(2).build()
+  const events = eventsBuilder.build()
+  const resources = { ...categoriesBuilder.buildResources(), ...eventsBuilder.buildResources() }
+  const fetchMap = { ...categoriesBuilder.buildFetchMap(), ...eventsBuilder.buildFetchMap() }
+
+  await dataContainer.setEvents(city, language, events)
+  await dataContainer.setCategoriesMap(city, language, categories)
+  await dataContainer.setLanguages(city, languages)
+  await dataContainer.setResourceCache(city, language, resources)
+  await dataContainer.setCities(cities)
+
+  return { languages, cities, categories, events, resources, fetchMap }
+}
+
+jest.mock('moment-timezone', () => {
+  const moment = jest.requireActual('moment-timezone')
+  const mockActualTz = moment.tz
+  moment.tz = jest.fn(mockActualTz)
```

Comment:
Didn't you propose not to use `moment.tz` anymore?

Reply:
It is stil used in the code. So far I did not change existing code.